gapii/cc/connection_header.h

```diff
@@ -48,6 +50,8 @@ class ConnectionHeader {
     uint32_t mVersion;                      // 2 or 3
     uint32_t mObserveFrameFrequency;        // non-zero == enabled. Version: 2+
     uint32_t mObserveDrawFrequency;         // non-zero == enabled. Version: 2+
+    uint32_t mStartFrame;                   // non-zero == Frame to start at. version 4+
+    uint32_t mNumFrames;                    // non-zero == Number of frames to capture. version 4+
```

Comment:
Just wonder if we should put the `mStartFrame` and `mNumFrames` to the end of the header struct?